src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/Microsoft.Azure.ServiceBus.Tests.csproj

```diff
@@ -11,28 +17,11 @@
 
   <ItemGroup>
     <Compile Remove="obj\**" />
+    <Compile Remove="bin\**" />
     <EmbeddedResource Remove="obj\**" />
+    <EmbeddedResource Remove="bin\**" />
     <None Remove="obj\**" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(IsTargetingNetFx)' == 'true'">
-    <Compile Remove="..\..\test\Microsoft.Azure.ServiceBus.UnitTests\ExpectedMessagingExceptionTests.cs" />
```

Comment:
After https://github.com/Azure/azure-sdk-for-net/pull/5756 we are running .NET framework tests so I suspect removing of these excludes might cause some of thse to fail.

Reply:
Those don't even point to the actual files as there is no ```test``` directory. It should be ```tests```.
Actually the correct url is ```src\SDKs\ServiceBus\data-plane\tests\Microsoft.Azure.ServiceBus.Tests\ExpectedMessagingExceptionTests.cs```

Reply:
Yeah, I tested in the full framework via VS test runner and Core via `dotnet test` - I've got the same test counts and same consistent passing results.    Not sure why they had them in here, but some of our stabilization work must have fixed whatever issue there was.

Reply:
Yeah if they don't exist then yes please do clean them up.

Reply:
Cleaned.   @chidozieononiwu:  I reverted your removal of the `MessageInteropEnd2EndTests`, since I fixed the refactoring miss on the scope naming.   Please let me know if that test is causing issues for you.